docs/new_services.rst

```diff
@@ -0,0 +1,41 @@
+.. _topics-new_services:
+
+New Services
+============
+
+"Service" refers generally to any process, possibly long-running, which you
+want to run on the test cluster.
+
+These can be services you would actually deploy
+(e.g., Kafka brokers, ZK servers, REST proxy) or processes used during testing
+(e.g. producer/consumer performance processes). You should also make each
+service class support starting a variable number of instances of the service so
+test code is as concise as possible.
+
+Each service is implemented as a class and should at least implement the following::
+
+    start_node(self, node) - start the service (possibly waiting to ensure it started successfully)
```

Comment:
We might be missing docstrings atm, but linking to the actual docs might be nice here